WooCommerce/UITestsFoundation/Screens/Orders/SingleOrderScreen.swift

```diff
@@ -38,20 +42,23 @@ public final class SingleOrderScreen: ScreenObject {
 
     @discardableResult
     public func verifySingleOrder(order: OrderData) throws -> Self {
+        let orderTotalPredicate = NSPredicate(format: "label CONTAINS %@", order.total)
+
         // Check that navigation bar contains order number
         let navigationBarTitles = app.navigationBars.map { $0.staticTexts.element.label }
         let expectedTitle = "#\(order.number)"
         XCTAssertTrue(navigationBarTitles.contains(where: { $0.contains(expectedTitle) }), "No navigation bar found with title \(expectedTitle)")
 
+        // Check order status and total
         let orderDetailTableView = app.tables["order-details-table-view"]
         orderDetailTableView.assertTextVisibilityCount(textToFind: order.status, expectedCount: 1)
-        orderDetailTableView.assertTextVisibilityCount(textToFind: order.total, expectedCount: 1)
+        XCTAssertTrue(app.otherElements.containing(orderTotalPredicate).element.exists)
```

Comment:
on a "paid" order there total is a `staticText` but on an order that is `awaiting payment`, that doesn't exist. there were a few ways to go around this:
1. update the mocks 
2. update this assert

i went with 2 because it was simpler. while i already updated the mock a bit to add the name, updating the price/total/states might mess up the other test depending on the mock.

the element tree to show why it passed for order 2201 but not order 3337:

order 2201:
<img width="736" alt="order 2201" src="https://user-images.githubusercontent.com/17252150/231946882-b5505d58-8f75-4684-86e5-4b5a7b9be2d5.png">

order 3337:
<img width="727" alt="order 3337" src="https://user-images.githubusercontent.com/17252150/231946934-8b6b3f2a-d4e3-4af9-81de-567f47bcc930.png">